predeploy.py

```diff
@@ -15,20 +14,7 @@
 if "DOKKU_SCALE" in os.listdir(app_path):
     shutil.copyfile(os.path.join(app_path, "DOKKU_SCALE"), "DOKKU_SCALE")
 
-
-file_list = rfiles if "-" in app_name and app_name.split("-")[0] == "dashr" else pyfiles
-
-for f in file_list:
+for f in pyfiles:
```

Comment:
👍 I think it's safe to :hocho: the R-related lines here, given that @tarzzz's fix uses the pre-predeploy.py CircleCI test flow to handle R app deployments.